CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Disable autocorrect on NoteComposer when running on macOS. [#1460](https://github.com/planetary-social/nos/issues/1460)
 
 ### Internal Changes
+- Reduced spammy "Failed to parse Follow" log messages
```

Comment:
Oh rats -- this is in the wrong section. I wish we could prevent this, or do something whenever we release to make sure we catch these sorts of things.